test/cloud_firestore_mocks_test.dart

```diff
@@ -348,7 +345,7 @@ void main() {
 
     final snapshot2 = await firestore.collection('users').document().get();
     expect(snapshot2, isNotNull);
-    expect(snapshot2.documentID.length, 20);
+    expect(snapshot2.documentID.length, greaterThanOrEqualTo(20));
```

Comment:
The auto-generated key length might not exact 20. In real Firestore, I got `/users/t1gVCkBrgMU9LjFRnrtu1FFjlir1/persons/DppyyRuUwQL7W6Qq7t8O` . The first key has 28 characters and the second has 20.